### ISSUE TITLE
Address Firefox console warnings.

### ISSUE THREAD

**Issue by johnnymatthews** [filecoin-project/filecoin-docs#1563]
There are _lots_ of warnings with unknown properties within Firefox CSS declarations:

<img width="1440" alt="Screen Shot 2023-01-12 at 1 19 47 PM" src="https://user-images.githubusercontent.com/9611008/212135772-ccf53ccd-3846-49dd-92ff-7d9b03d6664a.png">

The screenshot above was taken from a local instance of the Filecoin Docs website. While they're not breaking anything, we should find out where these warnings are coming from and resolve them.

**Comment by ElPaisano** [comment]
@johnnymatthews I can take this one

**Comment by ElPaisano** [comment]
@johnnymatthews 

This is confusing...

Not sure if updates to the site between this issue being opened and me investigating fixed some of these issues, but I'm on local `main` of my fork as of latest `main` for the main repo (https://github.com/ElPaisano/filecoin-docs/commit/3c44345ef6331bcfd50aaae6c7762b074b05f280), and am not seeing the same errors in the console

![image](https://user-images.githubusercontent.com/113373882/215589621-272b1237-3807-40ad-99f0-67affc485b41.png)

Based on the errors I saw and felt I could fix, made a few changes to _global.css in this [PR](https://github.com/filecoin-project/filecoin-docs/pull/1630)

**RE `Some cookies are misusing the recommended “SameSite“ attribute` error** -> (i think) this is saying that Cookies `_ga` and `_ga_H3D2P5EGFR` need to add the `SameSite=None` attributes if the site functionality depends on cookies will be sent in all contexts, i.e. in responses to both first-party and cross-site requests. Not sure how or where we set these cookies with the current infra set up, or if the site needs `SameSite=None` to function, but I feel like this is a browser specific thing that doesnt appear to affect functionality? I didn't see it locally in other browsers (chrome, brave)


**RE Errors in parsing css values or unknown class / element or ruleset ignored due to bad selector errors** -> Some of these errors occurring here appear to be in thirdparty (`bootstrap` theme) file located in node-modules
- `_forms.scss`
- `_reboot.scss`
Not sure how you treat these files in this project, but I've generally tried to avoid modifying modules / libraries that are liable to get updated and aren't maintained / forked directly by me. So, we could modify the files, or just ignore the warnings since they don't seem to be causing actual issues


Not sure on a few of these other ones, but I'll investigate further
